primer/brand
# FAQ a11y sign off updates

## Summary

A number of a11y changes to ensure full accessibility for the FAQ component.

## List of notable changes:

- **added** FAQ.Group to the FAQ component to provide accessible markup
- **updated** Accordion summary spacing so that the focus would not obscure the text


## What should reviewers focus on?

- Making sure each of the items within [this issue](https://github.com/github/primer/issues/1675) are addressed.

## Steps to test:

1. Open the FAQ component in CI-deployed preview environment
2. Go to FAQ story in Storybook
3. Verify that # behaves as described in the [following issue](https://github.com/github/primer/issues/1675).

## Supporting resources (related issues, external links, etc):

- closes https://github.com/github/primer/issues/1675

## Contributor checklist:


## Reviewer checklist:


## Screenshots:

> The spacing here between the heading and content is increased to allow for the focus border to not block the details text

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">

![image](https://github.com/primer/brand/assets/31220082/66a10183-9de8-45f2-822e-d54e27916cc5)

 </td>
<td valign="top">

![image](https://github.com/primer/brand/assets/31220082/8dd9a76d-d32d-4505-abf5-bd60835d5905)

</td>
</tr>
</table>
